pootle/apps/pootle_project/models.py

```diff
@@ -396,6 +395,10 @@ def clean(self):
                 _('"%s" cannot be used as a project code', self.code)
             )
 
+    def directory_exists(self):
```

Comment:
I agree with the idea. At least the method should be renamed to make it clear it is on disk.